config/samples/koku-metrics-cfg_v1alpha1_kokumetricsconfig.yaml

```diff
@@ -3,8 +3,15 @@ kind: KokuMetricsConfig
 metadata:
   name: kokumetricscfg-sample
 spec:
-  authentication: {}
-  packaging: {}
+  authentication:
```

Comment:
changing this sample will require us to update our make commands for local development.

Reply:
Moved a CR template into the testing file. Now we can make changes to this sample without breaking the Makefile.